packages/oracle-manager/contracts/utils/ChainlinkNodeLIbrary.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import "../storage/NodeFactoryStorage.sol";
+import "../interfaces/external/IAggregatorV3Interface.sol";
+
+library ChainlinkNodeLibrary {
```

Comment:
the filename is `ChainlinkNodeLIbrary`, but it should be `ChainlinkNodeLibrary` (if the filename doesnt match the name of its internal contract/library, it tends to cause problems)